Doc/whatsnew/3.10.rst

```diff
@@ -1298,6 +1298,14 @@ Add new function :func:`typing.is_typeddict` to introspect if an annotation
 is a :class:`typing.TypedDict`.
 (Contributed by Patrick Reader in :issue:`41792`)
 
+Subclasses of ``typing.Protocol`` which only have data variables declared
+will now raise a ``TypeError`` when checked with ``isinstance`` unless they
+are decorated with :func:`runtime_checkable`.  Previously, these checks
+passed silently.  Users should decorate their
+subclasses with the :func:`runtime_checkable` decorator in Python 3.10 and
+higher if they want runtime protocols.
```

Comment:
Even though it's only an error in 3.10+, users should use that decorator in all versions (it doesn't make anything worse to add it).
```suggestion
subclasses with the :func:`runtime_checkable` decorator
if they want runtime protocols.
```